dev/AppLifecycle/FileActivatedEventArgs.h

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#pragma once
+
+#include "ActivatedEventArgsBase.h"
+
+namespace winrt::Microsoft::ProjectReunion::implementation
+{
+    using namespace winrt::Windows::Foundation::Collections;
+    using namespace winrt::Windows::ApplicationModel::Activation;
+    using namespace winrt::Windows::Storage;
+
+    class FileActivatedEventArgs : public winrt::implements<FileActivatedEventArgs,
+        ActivatedEventArgsBase, IFileActivatedEventArgs>
+    {
+    public:
+        FileActivatedEventArgs([[maybe_unused]] const std::wstring contractData)
```

Comment:
seems like this isn't necessary?

Reply:
I assume you mean the [[maybe_unused]]?  Yeah, that is old.  I'll remove.  Thanks!